tribe/config/example_secrets.ini

```diff
@@ -1,21 +1,22 @@
-# This file contains values that you would like kept out of the repository 
-# for one reason or another.  It is specifically excluded in the .hgignore 
+# This file contains values that you would like kept out of the repository
+# for one reason or another.  It is specifically excluded in the .hgignore
```

Comment:
.hgignore -> .gitignore.

Reply:
Good catch! It was a leftover of mercurial migration.